src/effects/effectsmanager.h

```diff
@@ -13,6 +13,21 @@
 
 class EngineEffectsManager;
 
+// AdoptMetaknobValue determines how metaknobs behave when effects and presets are loaded.
+enum class AdoptMetaknobValue {
+    // When loading effects, never adjust the metaknob positions
+    KEEP_ALL,
+    // When loading effects presets, never adjust the top-most metaknob position.
```

Comment:
I have no idea what a "top-most metaknob" is. This is new terminology.

Reply:
would "Effect Unit Metaknob" be better?

Reply:
I still don't know what that is. Effects have metaknobs, not effect units.

Reply:
We are talking about "[EffectRack1_EffectUnit1_Effect2], meta". There is only one, at this layers, so I think we need to adjust the naming.

Currently we have this:
KEEP_ALL -> The  current meta knob position is adopted for the connected effect parameters, when loading effects and chain presets. 
KEEP_TOP -> When loading a chain preset, the metaknob and the parameters are changed to the saved values, including an out of sync state if a single parameter was adjusted before saving. If a single effect is loaded, the meta knob position is kept like "KEEP_ALL".  
LOAD_DEFAULT -> The saved metaknob adjust the parameters into a synced state, a saved individual parameter value is ignored. 

How about: 
KEEP
ADOPT_FROM_CHAIN_PRESET 
ADOPT_FROM_EFFECT

We may also consider to use two preferences option 
* Effect Load Behavior (Keep/Reset) 
* Effect Chain Preset Load Behaviour (Keep/Reset)

Do we really need the "KEEP_TOP (ADOPT_FROM_CHAIN_PRESET)" or can we use a common (Keep/Reset) value?